### PR TITLE
New jobs v1

### PR DESCRIPTION
**Pull request by CannonLock**

### CODE REVIEW

File: _data/jobs/research-systems-integrator.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+title: Research Cyberinfrastructure Specialist
+text: |
+  Come help us push the frontiers of human knowledge as a Research Cyberinfrastructure Specialist! The Center for High Throughput Computing (CHTC) at the University of Wisconsin–Madison is seeking candidates to work on large-scale distributed computing systems used by researchers across the world.
+  
+  As a Research Cyberinfrastructure Specialist, you’ll be working on projects ranging from integrating a variety of data sources for global distribution to developing reporting tools that provide insight to distributed cyberinfrastructure systems. Candidates of all experience levels, including recent graduates, are encouraged to apply!
+  
+  Email questions to [htcondor-jobs@cs.wisc.edu](htcondor-jobs@cs.wisc.edu).
+  
+  The application deadline is August 30, 2023.
```
Comment (brianhlin) [inline _data/jobs/research-systems-integrator.yaml:9]:
```suggestion
  The application deadline is September 5, 2023.
```

**Review by brianhlin — APPROVED** [review]
Some suggestions + a question but otherwise LGTM

File: _data/jobs/research-systems-integrator.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+button:
+  text: View Full Job Listing
+  href: https://go.wisc.edu/chtc-cyberinfrastructure-research-specialist-2023
+type: fulltime
+institution: Center for High Throughput Computing
+close_date:
```
[the remark this answers is not in the document]
Comment (CannonLock) [reply _data/jobs/research-systems-integrator.yaml:15]: It needs to be epoch time, I have added it.

**Comment by brianhlin** [comment]
LGTM, Cannon gave the thumbs up that we can ignore the CI failures.

**Pull request merged**